tests/js/spec/actionCreators/organization.spec.jsx

```diff
@@ -4,26 +4,20 @@ import OrganizationActions from 'app/actions/organizationActions';
 import ProjectActions from 'app/actions/projectActions';
 import TeamActions from 'app/actions/teamActions';
 import OrganizationStore from 'app/stores/organizationStore';
-import ProjectsStore from 'app/stores/projectsStore';
-import TeamStore from 'app/stores/teamStore';
 
 describe('OrganizationActionCreator', function () {
-  const detailedOrg = TestStubs.Organization({
-    teams: [TestStubs.Team()],
-    projects: [TestStubs.Project()],
-  });
+  const org = TestStubs.Organization();
+  delete org.teams;
+  delete org.projects;
```

Comment:
should we remove the properties `teams` and `projects` from the Organization Stub?

Reply:
Yeah we should.

I will be following up with more test cleanup